test/hpr_http_roaming_packet_SUITE.erl

```diff
@@ -0,0 +1,1588 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 06. Oct 2022 2:11 PM
+%%%-------------------------------------------------------------------
+-module(hpr_http_roaming_packet_SUITE).
```

Comment:
Rename to match the other test suites.

Reply:
@michaeldjeffrey 
I don't know what you are suggesting.

Reply:
Renamed two test SUITES.